web/tests/specs/oauth-provider.ts

```diff
@@ -9,7 +9,7 @@ async function reachTheProvider() {
     await ProvidersListPage.logout();
     await login();
     await ProvidersListPage.open();
-    await expect(await ProvidersListPage.pageHeader).toHaveText("Providers");
+    await expect(await ProvidersListPage.pageHeader()).toHaveText("Providers");
```

Comment:
Update this test to find according to our new Provider wizard (new card-radio interface ("cardio?"), different button identity.)